packages/shotwell.rb

```diff
@@ -0,0 +1,61 @@
+require 'package'
+
+class Shotwell < Package
+  description 'An open-source digital photo organiser for GNOME'
+  homepage 'https://wiki.gnome.org/Apps/Shotwell'
+  version '0.32.0'
+  license 'GPL-2.1+'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://download.gnome.org/sources/shotwell/0.32/shotwell-0.32.0.tar.xz'
+  source_sha256 '1772b2fa1e7a5479d528028bc9a303f2855e1d959a5b3c917922e63a98a90b19'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/shotwell/0.32.0_armv7l/shotwell-0.32.0-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/shotwell/0.32.0_armv7l/shotwell-0.32.0-chromeos-armv7l.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/shotwell/0.32.0_x86_64/shotwell-0.32.0-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '088c46df2ab44dd0279c06cacc8b76a6fc2aef0463616f0df531b8476fa45082',
+     armv7l: '088c46df2ab44dd0279c06cacc8b76a6fc2aef0463616f0df531b8476fa45082',
+     x86_64: 'a998826a68da79628baabdc79c243038762dd32e2b8d1b3e14a46a580e55adf2'
+  })
+
+  depends_on 'vala' => :build
+  depends_on 'cairo' # R
+  depends_on 'gcr' # R
+  depends_on 'gdk_base' # R
+  depends_on 'gdk_pixbuf' # R
+  depends_on 'gexiv2' # R
+  depends_on 'glib' # R
+  depends_on 'gstreamer' # R
+  depends_on 'gtk3' # R
+  depends_on 'libgee' # R
+  depends_on 'libgphoto' # R
+  depends_on 'libgudev' # R
+  depends_on 'libportal' # R
+  depends_on 'libraw' # R
+  depends_on 'libsecret' # R
+  depends_on 'libsoup' # R
+  depends_on 'libwebp' # R
+  depends_on 'libxml2' # R
+  depends_on 'json_glib' # R
+  depends_on 'sqlite' # R
+  depends_on 'webkit2gtk' # R
+  depends_on 'sommelier'
```

Comment:
Doesn't need sommelier since it has gtk3.

Reply:
The idea here is to make sure sommelier is installed last to make sure the post install instructions are not missed.

Reply:
BTW, I could probably exclude a lot of the dependencies here that are installed implicitly. ;)

Reply:
> The idea here is to make sure sommelier is installed last so the post install instructions are not missed.

Remember how sommelier's post-install  is already set to be shown last because of code in `crew`?

Reply:
> BTW, I could probably exclude a lot of the dependencies here that are installed implicitly. ;)

I'm of course only talking about the non-library dependencies. 😅🫡

Reply:
OK, fair enough.  Removed.